spec/rails_helper.rb

```diff
@@ -13,6 +13,10 @@
 require 'support/request_helpers'
 require 'webmock/rspec'
 require 'paper_trail/frameworks/rspec'
+require 'capybara/rails'
```

Comment:
Nice! I like Capybara! I like the way integration specs look with this gem. However, I do think this is a big change in the way we've been testing stuff so far. And I believe we should have a consistent method for testing across the team, I believe it's a big enough subject to discuss in our daily check-in.